internal/host/hostcommands/install_cmd.go

```diff
@@ -252,7 +252,8 @@ func (i *installCmd) getDiskUnbootableCmd(ctx context.Context, host models.Host)
 	formatCmds := ""
 	for _, disk := range inventory.Disks {
 		isFcIscsi := strings.Contains(disk.ByPath, "-fc-") || strings.Contains(disk.ByPath, "-iscsi-")
-		if disk.Bootable && !isFcIscsi && !disk.IsInstallationMedia {
+		isMmcblk := strings.Contains(disk.ByPath, "mmcblk") //mmc devices should be treated as removable
```

Comment:
Consider moving this logic to the agent, which would set removable to True if:
1. The disk is actually removable, or
2. The device starts with mmcblk

Reply:
Similar logic are placed here. Moving it to the agent will cause maintenance issues

Reply:
OK